packages/meditrak-app/app/sync/Synchroniser.js

```diff
@@ -246,61 +250,50 @@ export class Synchroniser {
    */
   async pushChanges(changesSyncJson) {
     const startTime = new Date().getTime();
-    try {
```

Comment:
Removed all the redundant try/catches. ESlint told me to

Reply:
Yes thank you for this!